src/test/java/it/gov/pagopa/idpay/transactions/service/RewardTransactionServiceImplTest.java

```diff
@@ -0,0 +1,78 @@
+package it.gov.pagopa.idpay.transactions.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.gov.pagopa.idpay.transactions.dto.mapper.RewardTransactionMapper;
+import it.gov.pagopa.idpay.transactions.model.RewardTransaction;
+import it.gov.pagopa.idpay.transactions.repository.RewardTransactionRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.support.MessageBuilder;
+import reactor.core.publisher.Flux;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+class RewardTransactionServiceImplTest {
+    @Test
+    void findByIdTrxIssuer() {
+        // Given
+        RewardTransactionRepository rewardTransactionRepository = Mockito.mock(RewardTransactionRepository.class);
+        RewardTransactionMapper rewardTransactionMapper = Mockito.mock(RewardTransactionMapper.class);
+        ErrorNotifierService errorNotifierService = Mockito.mock(ErrorNotifierService.class);
+        ObjectMapper objectMapper = Mockito.mock(ObjectMapper.class);
+
+        RewardTransactionService rewardTransactionService= new RewardTransactionServiceImpl(rewardTransactionRepository, rewardTransactionMapper,errorNotifierService, objectMapper);
```

Comment:
instead of repeat this at each test, you could put these on class level and perform one of the following:

* set as final the mocks and use the constructor to Mock it
* Annotate the class level mocks with @Mock, annotate the class with @ExtendWith(MockitoExtension.class)

then use a @PostConstruct or @BeforeEach method to create the Service to test